src/index.ts

```diff
@@ -1518,47 +1519,38 @@ export default class Client {
    *
    * @param {string} projectId
    * @param {string} environmentTypeId
-   * @param {ProjectAccess} access
+   * @param {string} accessId
+   * @param {AccessRole} role
    *
    * @returns {Promise} Promise that return an access object.
    */
-  async updateEnvironmentTypeAccess(projectId: string, environmentTypeId: string, access: ProjectAccess) {
-    return entities.EnvironmentType.updateAccess(
-      projectId,
-      environmentTypeId,
-      access
-    );
+  async updateEnvironmentTypeAccess(params: UpdateAccessParams) {
+    return entities.EnvironmentType.updateAccess(params);
   }
   /**
    * Delete project environment types accesses
    *
    * @param {string} projectId
    * @param {string} environmentTypeId
-   * @param {ProjectAccess} access
+   * @param {string} accessId
    *
    * @returns {Promise} Promise that return a Result.
    */
-  async deleteEnvironmentTypeAccess(projectId: string, environmentTypeId: string, access: ProjectAccess) {
-    return entities.EnvironmentType.deleteAccess(
-      projectId,
-      environmentTypeId,
-      access
-    );
+  async deleteEnvironmentTypeAccess(params: DeleteAccessParams) {
```

Comment:
The parameters of the function have changed but the comment has not been updated.